mmv1/third_party/terraform/resources/resource_container_cluster.go.erb

```diff
@@ -537,6 +537,28 @@ func resourceContainerCluster() *schema.Resource {
 										ForceNew:    true,
 										Description: `The Customer Managed Encryption Key used to encrypt the boot disk attached to each node in the node pool.`,
 									},
+									"shielded_instance_config": &schema.Schema{
+										Type:     schema.TypeList,
+										Optional: true,
+										Description: `Shielded Instance options.`,
+										MaxItems: 1,
+										Elem: &schema.Resource{
+											Schema: map[string]*schema.Schema{
+												"enable_secure_boot": {
```

Comment:
With all optional children, `shielded_instance_config` could be an empty block. Is there any significance in that? If not, could we add `AtLeastOneOf` to the children?

Reply:
AFAIK there's no significance to having an empty block, so that would make sense. Should I update the schema in [node_config.go.erb](https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/utils/node_config.go.erb#L308) as well so they match?